locale/en_US/submission.xml

```diff
@@ -503,7 +503,7 @@
 	<message key="submission.upload.uploadFiles">Upload Files</message>
 	<message key="submission.upload.response.description">After reviewing the assigned file, record any comments on the file below, and/or upload an annotated copy of the file (or another file).</message>
 	<message key="submission.upload.productionReady">Upload a Production Ready File</message>
-	<message key="submission.upload.proof">Upload a Proof</message>
+	<message key="submission.upload.proof">Upload a File Ready for Publication</message>
```

Comment:
When changing the translation for a key, the key should be removed from other locale/languages files, in order for it to be considered for translation again, I think.